### ISSUE TITLE
Create a Data Loader Class with Pytorch Lightning

### ISSUE THREAD

**Issue by lucapinello** [pinellolab/DNA-Diffusion#12]
For the first version make sure the latest version provided by @meuleman   here: 

https://www.meuleman.org/research/synthseqs/#material

Specifically, we have the following datasets available:

**training set: 160k sequences, 10k per NMF component (chr3-chrY, .csv.gz)
validation set: 16k sequences, 1k per NMF component (chr2 only, .csv.gz)
test set: 16k sequences, 1k per NMF component (chr1 only) .csv.gz)**

Each of these contains the genomic locations (human genome assembly hg38, first 3 columns) of accessible genome elements, their majority NMF component (column: component, see legend figure above) as well as their 200bp nucleotide sequence (column: raw_sequence).

Column    Example    Description
seqname    chr16    Chromosome
start    68843660    Start position
end    68843880    End position
DHS_width    220    Width of original DHS
summit    68843790    Position of center of mass of DHS
total_signal    122.770678    Sum of DNase-seq signal across biosamples
numsamples    61    Number of biosamples with the DHS
raw_sequence    GAGGCATTG…    200bp extracted nucleotide sequence
component    1    Dominant DHS Vocabulary component
proportion    0.767371514    Proportion of NMF loadings in this component

**Comment by lucapinello** [comment]
The important columns for the training are: 

raw_sequence
component 1 

You can think these two as the X (raw_sequence) and y (component)in a classic label classification problems, the other should be also provided/accessible by the Data Loader.

**Comment by codeghees** [comment]
How can I assign myself to the issue?

**Comment by ssenan** [comment]
I created an initial pull request for what the dataset loader might look like. I'd appreciate any feedback regarding the formatting and setup of the loader, so that I can alter it to suit our needs better
